charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet.flags

```diff
@@ -16,6 +16,9 @@
 {{- if semverCompare "< 1.14" .Values.kubernetes.version }}
 --feature-gates=SupportPodPidsLimit=true
 {{- end }}
+{{- if .Values.worker.cri }}
+--feature-gates=CRIContainerLogRotation=true
```

Comment:
Why the cri log rotation is enabled? Won't be a conflict between logrotate and cri log rotation such as the one described in https://github.com/gardener/gardener-extension-os-suse-jeos/issues/6?

Reply:
Docker is not used as container runtime when the .Values.worker.cri exist.
According to what i saw, logrotate is responsible for the log rotation of Docker containers.

Reply:
How we make sure with cri log rotation that logs older than 2 weeks are deleted? See https://github.com/gardener/gardener/issues/653.

Reply:
Well, need to be checked about 2 weeks
The log rotation can be controlled by:
--container-log-max-size and --container-log-max-files

Reply:
The easy way to fix this is to not use the kubelet log rotation option, but rather use logrotated for containerd, similar to how this is done for Docker. See #664 - it's basically adding a logrotated configuration file instead of specifying these two options.

@ialidzhikov , @rfranzke : would it be OK to merge this PR and then handle the additional requirements for log rotation in a future PR?

Reply:
@liorokman Your proposal sounds good, though, I'm afraid we have to do it in this PR. Once `containerd` is exposed in the `Shoot` manifest as option (which it will be with this PR) people expect similar experience wrt standards as before with Docker (#653). We already have some logrotate configuration, can this be enhanced to also work for the `containerd` case?

Reply:
Done